ui/scripts/instances.js

```diff
@@ -2948,7 +2969,7 @@
                                     }
 
                                     $.ajax({
-                                        url: createURL('updateVmNicIp'),
+                                        url: createURL('updateVmNicIpMtu'),
```

Comment:
@DaanHoogland 

As for reverting the API changes, I guess this needs to be reverted also?

Reply:
yes @holgerstolzenberg